gradle.properties

```diff
@@ -1,9 +1,11 @@
 version=10.5-SNAPSHOT
 riderBuildVersion=RD-2022.3.1
 clionBuildVersion=CL-2022.3.1
+resharperBuildVersion=RD-2024.1-SNAPSHOT
 intellijBuildVersion=IC-2022.3.1
 # Should be the same as the one embedded into SonarLint Core
 protobufVersion=3.25.1
 omnisharpVersion=1.39.10
 org.gradle.jvmargs=-Xmx8G -XX:MaxMetaspaceSize=512M
-org.gradle.caching=true
\ No newline at end of file
+org.gradle.caching=true
+kotlin.stdlib.default.dependency = false
```

Comment:
Is this line related to CLion? I think I added it myself but I don't remember, if you know, adding a comment telling why it's needed would help

Reply:
You added it and I don't remember the reason as well, seems like no difference. IIRC it was due to some warnings during build time, but might be wrong

Reply:
Could it be related to the `LinkageError` with the `GotItTooltip` we had originally? I also found this [doc](https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#kotlin-standard-library). If it's not needed, then we can drop it